File: stone_processor/src/main/java/com/github/klee0kai/stone/exceptions/ImplementMethodStoneException.java
package com.github.klee0kai.stone.exceptions;

import javax.lang.model.element.Element;

public class ImplementMethodStoneException extends StoneException {

    public ImplementMethodStoneException(String message, Throwable cause) {
        super(message, cause);
    }

    public ImplementMethodStoneException(String message, Throwable cause, Element element) {
        super(message, cause, element);
    }

}

File: stone_ksp/src/commonMain/kotlin/com/github/klee0kai/thekey/stone/ksp/exceptions/ImplementMethodStoneException.kt
package com.github.klee0kai.thekey.stone.ksp.exceptions

import com.google.devtools.ksp.symbol.KSNode

class ImplementMethodStoneException(
    message: String? = null,
    cause: Throwable? = null,
    element: KSNode? = null,
) : StoneException(
    message = message,
    cause = cause,
    element = element,
)
